doc/user_manual/step.tex

```diff
@@ -103,6 +103,9 @@ \subsection{SingleRun}
 in the \xmlNode{RunInfo} block, under the \xmlNode{Sequence} node. If the name
 of this \textbf{Step} is not listed in the  \xmlNode{Sequence} block, its action is not
 going to be performed.
+\item \xmlAttr{repeatFailureRuns}, \xmlDesc{optional integer attribute}, this optional 
+attribute could be used to set a certain number of repeations that need to be performed 
```

Comment:
I think you mean "repetitions".
"repeations" -> "repititions" ->  "repetitions"